financial-connections/res/values/strings.xml

```diff
@@ -46,6 +46,8 @@
     <string name="data_accessible_callout_no_business"><annotation bold="accessible_data">Data accessible to this business: </annotation>%1$s.<annotation clickable="learn_more"> Learn more</annotation></string>
```

Comment:
This is debatable and likely doesn't cause an issue unless we do things like making the link underline, but the annotation itself contains a space:

```
<annotation clickable="learn_more"> Learn more</annotation>
```

so if annotation _did_ underline, the space would have an underline

(again, likely not a legit issue, just calling it out in case there could be issues)

Reply:
It's a good point, fixed it! https://github.com/stripe/stripe-android/pull/6375#:~:text=Updates%20spaces.-,e4133f5,-Merge%20state